pytao/util/parsers.py

```diff
@@ -1,6 +1,65 @@
 import numpy as np
 
 
+
+
+data_d_cols =  ['ix_d1',
+ 'data_type',
+ 'merit_type',
+ 'ele_ref_name',
+ 'ele_start_name',
+ 'ele_name',
+ 'meas_value',
+ 'model_value',
+ 'design_value',
+ 'useit_opt',
+ 'useit_plot',
+ 'good_user',
+ 'weight',
+ 'exists']
+data_d_types = [int, str, str, str, str, str, float, float, float, bool, bool, bool, float, bool]
```

Comment:
Just a suggestion to move the two "constants" into the `parse_data_d_array` function or at least add some comments around them that they are used for the `parse_data_d_array` function.

Reply:
Okay, I changed it a little.